docgen/layouts/api.pug

```diff
@@ -0,0 +1,35 @@
+mixin nav(currentPath, nav)
+  ul
+    for navItem in nav
+      - var className = navItem.path === currentPath ? 'navItem--active' : 'navItem';
+      li(class=className)
+        a(href=`/${navItem.path}`)=navItem.file.title
```

Comment:
You may have to use the publicPath helper otherwise it will fail on /instantsearch.js/react/ no?